homework/index.js

```diff
@@ -29,23 +24,149 @@
     return elem;
   }
 
+  function convertDateTime(dateTimeTxt) {
+    const dateTime = new Date(dateTimeTxt);
+    return dateTime.toLocaleString();
+  }
+
+  function sortRepositoriesByName(a, b) {
+    return a.name.localeCompare(b.name);
+  }
+
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    ul.textContent = '';
+    const repoList = createAndAppend('li', ul, { class: 'repo-list' });
+    const repositoryName = createAndAppend('p', repoList, {
+      text: ' Repository: ',
+      class: 'repo-details',
+    });
+
+    const description = createAndAppend('p', repoList, {
+      text: ' Description: ',
+      class: 'repo-details',
+    });
+
+    const forkedNumbers = createAndAppend('p', repoList, {
+      text: ' Forks: ',
+      class: 'repo-details',
+    });
+
+    const updatedTime = createAndAppend('p', repoList, {
+      text: ' Updated: ',
+      class: 'repo-details',
+    });
+
+    createAndAppend('a', repositoryName, {
+      text: repo.name,
+      href: repo.html_url,
+      target: '_blank',
+    });
+
+    if (repo.description === null) {
```

Comment:
This effectively only renders a description when there is no description (it then renders 'No Description Added Yet'). In all other cases the Description field remains empty. Did you test and visually inspect your application before submitting the PR? This `if` statement is unneeded.

Reply:
when I shortened if else condition to make the code DRYer, I forgot to delete if statement. when i check the web-page visually i didnt recognize that description part was missing.